helm/charts/infra/README.md

```diff
@@ -0,0 +1,70 @@
+
+```bash
+RELEASE_NAME=infra
+RELEASE_NAMESPACE=$(helm status $RELEASE_NAME -o json | jq -r .namespace)
+HELM_ARGS=.. # additional arguments to helm, e.g. --set or --values
+
+### helm repo update infrahq
+helm -n $RELEASE_NAMESPACE upgrade $RELEASE_NAME infrahq/infra $HELM_ARGS
+```
+
+Once upgraded, verify the deployment with `infra login` or the Infra UI. Once verified, the server persistent volume claim should be disabled, `server.persistence.enabled=false`, to fully remove the SQLite database and old database encryption key.
+
+## Rollback
+
+If the upgrade is unsuccessful, the deployment can be rolled back using Helm. The upgrade can be attempted again or you can contact us through GitHub for additional assistance.
+
+```bash
+RELEASE_NAME=infra
+RELEASE_NAMESPACE=$(helm status $RELEASE_NAME -o json | jq -r .namespace)
+
+helm -n $RELEASE_NAMESPACE rollback $RELEASE_NAME
+```
+
+### 0.14.4
+
+Infra chart version 0.20.0 introduces two backwards incompatible changes. The first is the migration to PostgreSQL as the default backend database from SQLite. The second is the migration of the database encryption key from within the server PVC to a Kubernetes secrets. This guide will detail the required steps to successfully migrate an Infra deploy to 0.20.0.
+
+#### Database Encryption Key
+
+This change requires manual migration before performing the upgrade. If your deployment uses a database encryption key stored outside of Kubernetes, i.e. if your Helm values defines `server.config.dbEncryptionKey`, you can ignore this section.
+
+The following commands creates a Kubernetes secret with the necessary labels and annotations to masquerade as a Helm-managed resource. It seeds this secrets with the current encryption key such that it’s available to Infra after upgrade. 
+
+These commands assume the Helm release is called `infra`. Update the `RELEASE_NAME` variable if necessary with the correct release name. 
+
+```bash
+RELEASE_NAME=infra
+RELEASE_NAMESPACE=$(helm status $RELEASE_NAME -o json | jq -r .namespace)
+FULL_NAME=$(kubectl -n $RELEASE_NAMESPACE get deployment -l app.kubernetes.io/instance=$RELEASE_NAME -l app.infrahq.com/component=server -o name | cut -f2 -d/)
+
+ENCRYPTION_KEY=$(mktemp)
+kubectl -n $RELEASE_NAMESPACE exec -i deployment/$FULL_NAME -- cat /var/lib/infrahq/server/sqlite3.db.key >$ENCRYPTION_KEY
+kubectl -n $RELEASE_NAMESPACE create secret generic $FULL_NAME-encryption-key --from-file=key=$ENCRYPTION_KEY
+kubectl -n $RELEASE_NAMESPACE annotate secret $FULL_NAME-encryption-key meta.helm.sh/release-name=$RELEASE_NAME meta.helm.sh/release-namespace=$RELEASE_NAMESPACE 
+kubectl -n $RELEASE_NAMESPACE label secret $FULL_NAME-encryption-key app.kubernetes.io/managed-by=Helm
+rm $ENCRYPTION_KEY
+```
+
+#### PostgreSQL Database
+
+This change is automated and should not require manual migration. If your deployment uses an external database, i.e. if your Helm values defines `server.config.dbHost`, you can ignore this section.
```

Comment:
Why not put the part above for checking `server.persistence.enabled=false` here instead of above?

Reply:
The `server.persistence.enabled=false` step is a post-upgrade step to remove the server pvc and can only be set once upgraded and migrated. Doing so before will cause data loss. Added some more heading for clarity